src/main/java/duke/command/OnCommand.java

```diff
@@ -0,0 +1,73 @@
+package duke.command;
+
+import java.time.temporal.Temporal;
+
+import duke.datetime.DateTime;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.tasklist.TaskList;
+
+/**
+ * Represents an on command that is entered by the user to check what tasks are there on a specific day.
+ */
+public class OnCommand extends Command {
```

Comment:
Maybe can think of a better name, maybe a verb, that is more description of the action.

Reply:
Alright! Thanks for the suggestion 👍